src/pages/index.tsx

```diff
@@ -1,166 +1,28 @@
-import PupilTab from "@/components/GenericPagesComponents/PupilTab";
-import AiTab from "@/components/GenericPagesComponents/AiTab";
-import {
-  SerializedBlogPostPreview,
-  blogToPostListItem,
-} from "@/components/GenericPagesViews/BlogIndex.view";
-import {
-  SerializedWebinarPreview,
-  webinarToPostListItem,
-} from "@/components/GenericPagesViews/WebinarsIndex.view";
-import { HomePage } from "@/common-lib/cms-types";
-import { PostListItemProps } from "@/components/SharedComponents/PostListItem";
-import curriculumApi2023, {
-  TeachersHomePageData,
-} from "@/node-lib/curriculum-api-2023";
-import BlogAndWebinarList from "@/components/GenericPagesComponents/BlogAndWebinarList";
-import { getAndMergeWebinarsAndBlogs } from "@/utils/getAndMergeWebinarsAndBlogs";
+import Teachers, {
+  getStaticProps as getStaticPropsTeachers,
+  TeachersHomePageProps,
+} from "@/pages/teachers";
 
-export type TeachersHomePageProps = HomePageProps & {
-  curriculumData: TeachersHomePageData;
-};
-
-export type SerializedPost =
-  | ({ type: "blog-post" } & SerializedBlogPostPreview)
-  | ({ type: "webinar" } & SerializedWebinarPreview);
-
-export type HomePageProps = {
-  pageData: HomePage;
-  posts: SerializedPost[];
-};
-
-export type HomePageTab = "teachers" | "curriculum" | "pupils" | "ai";
-const isHomePageTab = (u: unknown): u is HomePageTab =>
-  u === "teachers" || u === "curriculum" || u === "pupils" || u === "ai";
-
-export const postToPostListItem = (post: SerializedPost): PostListItemProps => {
-  return post.type === "blog-post"
-    ? blogToPostListItem(post)
-    : webinarToPostListItem(post);
-};
-
-export const sortByDate = (a: { date: Date }, b: { date: Date }) => {
-  return b.date.getTime() - a.date.getTime();
-};
-
-const Teachers: NextPage<TeachersHomePageProps> = (props) => {
-  const { curriculumData } = props;
-  const posts = props.posts.map(postToPostListItem);
-  const [current, setCurrent] = useState<HomePageTab | undefined>();
-  const blogListProps = usePostList({ items: posts, withImage: true });
-  const { track } = useAnalytics();
-  const newsletterFormProps = useNewsletterForm({
-    onSubmit: track.newsletterSignUpCompleted,
-  });
+const Home: NextPage<TeachersHomePageProps> = (props) => {
+  const router = useRouter();
 
   useEffect(() => {
-    const tabHash = window.location.hash;
-    const tabName = tabHash.slice(1);
-
-    if (isHomePageTab(tabName) && current !== tabName) {
-      setCurrent(tabName as HomePageTab);
-    } else if (current === undefined) {
-      // default value when first loading the page or invalid hash
-      setActiveTab("teachers");
+    // clientside redirect for old tabs implementation
+    if (window.location.href.includes("#pupils")) {
+      router.push("/pupils");
+    } else if (window.location.href.includes("#curriculum")) {
+      router.push("/curriculum");
+    } else if (window.location.href.includes("#ai")) {
+      router.push("/ai");
```

Comment:
would adding these as redirects in the next.config not work?

Reply:
nope . unfortunately #s aren't recognised by next config. This is the only way to do it on pages router

Reply:
ahh I see, that's annoying 🫠